src/screens/OMPSwitch/MerchantSwitch.res

```diff
@@ -162,7 +162,7 @@ let make = () => {
   let customHRTagStyle = "border-t border-blue-830"
   let customPadding = "py-1 w-full"
   let customStyle = "w-56 text-gray-200 bg-blue-840 dark:bg-black hover:bg-popover-background-hover hover:text-gray-100 !w-full"
-
+  let customScrollStyle = "max-h-72 overflow-scroll scrollbar-thin px-1 pt-1 "
```

Comment:
`scrollbar-thin` class is not needed as you have already mentioned it in `sidebarScrollbarCss` in SelectBox

Reply:
Removed the redundant scrollbar-thin class.